ci/cloudbuild/substra-backend.yaml

```diff
@@ -0,0 +1,18 @@
+steps:
+- name: 'gcr.io/cloud-builders/git'
+  args: ['clone', 'https://github.com/SubstraFoundation/substra-backend.git', '--depth', '1', '--branch', '${_BRANCH}']
+- name: 'gcr.io/kaniko-project/executor:latest'
+  args:
+  - --destination=gcr.io/$PROJECT_ID/substra-backend:${_COMMIT}
+  - --cache=true
+  - --dockerfile=substra-backend/docker/substra-backend/Dockerfile
+  - --context=substra-backend
+  - --cache-ttl=${_KANIKO_CACHE_TTL}
+tags:
+- ${_BUILD_TAG}
+substitutions:
+    _BUILD_TAG: "substra-tests"
+    _COMMIT: "HEAD"
+    _BRANCH: "master"
```

Comment:
just trying to understand, why we don't use ${_BRANCH} here?

Reply:
The `substitutions` section just defines the variables of our script. This section is mandatory, and the values you see here are the (mandatory) default values.

Reply:
thank's @AurelienGasser